File: BlackFireIM/Example/Lib/MSIMSDK-UIKit/Base/MSHelper.m
//
//  MSHelper.m
//  BlackFireIM
//
//  Created by benny wang on 2021/5/27.
//

#import "MSHelper.h"
#import <SVProgressHUD.h>


@implementation MSHelper

+ (void)showToast
{
    [SVProgressHUD show];
}

+ (void)showToastString:(NSString *)text
{
    [SVProgressHUD showInfoWithStatus:text];
}

+ (void)showToastSucc:(NSString *)text
{
    [SVProgressHUD showSuccessWithStatus:text];
}

+ (void)showToastFail:(NSString *)text
{
    [SVProgressHUD showErrorWithStatus:text];
}

+ (void)dismissToast
{
    [SVProgressHUD dismiss];
}


@end

File: Swift/MSIM-Swift/Lib/MSIMSDK-UIKit/Base/MSHelper.swift
//
//  MSHelper.swift
//  MSIM-Swift
//
//  Created by benny wang on 2021/7/28.
//

import Foundation
import SVProgressHUD

class MSHelper {
    
    public class func showToast() {
        
        SVProgressHUD.show()
    }
    
    public class func showProgress(progress: Float,text: String?) {
        
        SVProgressHUD.showProgress(progress, status: text)
    }
    
    public class func showToastWithText(text: String) {
        
        SVProgressHUD.setMinimumDismissTimeInterval(3)
        SVProgressHUD.showInfo(withStatus: text)
    }
    
    public class func showToastSuccWithText(text: String) {
        
        SVProgressHUD.setMinimumDismissTimeInterval(3)
        SVProgressHUD.showSuccess(withStatus: text)
    }
    
    public class func showToastFailWithText(text: String) {
        
        SVProgressHUD.setMinimumDismissTimeInterval(3)
        SVProgressHUD.showError(withStatus: text)
    }
    
    public class func dismissToast() {
        
        SVProgressHUD.dismiss()
    }
}
